perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -423,24 +423,33 @@ sub inspect : Private {
             push @{ $c->stash->{errors} }, _('Invalid location. New location must be covered by the same council.');
         }
 
-        if ($permissions->{report_inspect} || $permissions->{report_edit_category}) {
-            $c->forward( '/admin/report_edit_category', [ $problem ] );
-
-            # The new category might require extra metadata (e.g. pothole size), so
-            # we need to update the problem with the new values.
-            my $param_prefix = lc $problem->category;
-            $param_prefix =~ s/[^a-z]//g;
-            $param_prefix = "category_" . $param_prefix . "_";
-            my @contacts = grep { $_->category eq $problem->category } @{$c->stash->{contacts}};
-            $c->forward('/report/new/set_report_extras', [ \@contacts, $param_prefix ]);
-        }
 
         # Updating priority must come after category, in case category has changed (and so might have priorities)
```

Comment:
This comment seems quite relevant, as you've moved the the category change after it :) I guess this should move down under the bit you've moved.

Reply:
I confess I don't really understand what this comment means as nothing in here seems dependant on the report category.

Reply:
`$problem->response_priorities` uses `$self->category` to locate the appropriate priorities.